internal/uvm/create_lcow.go

```diff
@@ -354,7 +357,7 @@ func CreateLCOW(ctx context.Context, opts *OptionsLCOW) (_ *UtilityVM, err error
 	}
 
 	kernelArgs += fmt.Sprintf(" nr_cpus=%d", opts.ProcessorCount)
-	kernelArgs += ` pci=off brd.rd_nr=0 pmtmr=0 -- ` + initArgs
+	kernelArgs += ` brd.rd_nr=0 pmtmr=0 -- ` + initArgs
```

Comment:
Did we ever verify boot changes here? Do we want to make pci=off a setting only for gpu scenarios?

Reply:
We have not yet, still a WIP there :)

Reply:
I think we should just make this configurable and non-default until we test this

Reply:
Basically a `lopt` for vPCIEnabled (default false)

Reply:
Done.

Reply:
Thanks!